Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionMenuItemCocoa.mm

```diff
@@ -325,7 +325,7 @@ + (BOOL)usesUserKeyEquivalents
 
     // iOS does not support sub-menus that are disabled or hidden, so return a normal action in that case.
     if (submenuItems().isEmpty() || !isEnabled() || !isVisible()) {
-        auto *action = [UIAction actionWithTitle:processedTitle image:icon(CGSizeMake(20, 20)) identifier:nil handler:makeBlockPtr([this, protectedThis = Ref { *this }, contextParameters](UIAction *) mutable {
+        auto *action = [UIAction actionWithTitle:processedTitle image:(icon({ 20, 20 })->image().get()) identifier:nil handler:makeBlockPtr([this, protectedThis = Ref { *this }, contextParameters](UIAction *) mutable {
```

Comment:
I think this is causing a crash on iOS, since you don't null check `icon({ 20, 20 })`.
```suggestion
        UIImage *image;
        if (RefPtr icon = this->icon({ 20, 20 }))
            image = icon->image().get();
        auto *action = [UIAction actionWithTitle:processedTitle image:image identifier:nil handler:makeBlockPtr([this, protectedThis = Ref { *this }, contextParameters](UIAction *) mutable {
```

We should consider a helper function for this, so it is easy to get a `nil` for a `null` `WebCore::Icon`.

```
static inline CocoaImage *toCocoaImage(const RefPtr<WebCore::Icon>& icon)
{
    return icon ? icon->image().get() : nil;
}
```

That needs to return `RetainPtr<CocoaImage>` if you out it in `WebCore/Icon.h`, since that code isn't ObjC ARC.